app/Rules/README.md

```
@@ -224,4 +224,8 @@ class UpdateAlbumRequest extends BaseApiRequest
 }
 ```
 
-This validation rule system ensures that Lychee maintains high data quality and security standards while providing clear, maintainable validation logic throughout the application.
\ No newline at end of file
+This validation rule system ensures that Lychee maintains high data quality and security standards while providing clear, maintainable validation logic throughout the application.
+
+---
+
+*Last updated: August 14, 2025*
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)